### DISCUSSION TITLE
Help with hack rom conversion

### DISCUSSION THREAD

**Discussion by lzibeti** [city41/neosdconv#15] [Q&A]
Hello, I'm trying to convert a modified ROM from KOF96 (The King of Fighters 96 Anniversary Edition), it has new characters and moves but everything new is defective in the image, full of glitch (the rest works fine).

I tried both with your converter and the Terraonion but both end up giving the same problem. I believe the problem occurs in the C files

The hack has 12 C roms of 4MB each and when trying to convert with your converter it gets lost and puts the duplicate c10, c11 and c12 files, first after c1 and then at the end, after c9.

![image](https://user-images.githubusercontent.com/58938473/159080461-100102fa-b0a0-4556-a51b-655b20a5cbd6.png)

A friend of mine converted these 12 C files into 8 C files of 6MB each, even running on NeoRageX that way, but even so the game converts with errors in the graphics.

![image](https://user-images.githubusercontent.com/58938473/159081186-65cbac5c-5987-496c-8606-8a9b496bd29b.png)

Can someone help me in trying to convert this hack to play it on Neo Geo AES using Neo SD?

Thank you very much.

**Comment by city41** [comment]
The way it's doing `c1` then `c10` does look like a bug in neosdconv. When I get some spare cycles I'll look into that.

But to be honest if it doesn't work with NeoBuilder, there's not much hope of it working in neosdconv either. Have you tried asking in the TerraOnion discord? They are usually pretty helpful.

If you can get it working in NeoBuilder, I'd probably be able to get it working in neosdconv, but the problem is if it doesn't work in NB, then it could mean something fundamental about the game is incompatible with the NeoSD and so I'd be trying to fix something that is not possible to fix. For example the new Cabal port just doesn't work on the NeoSD due to how it was made.
